src/main/java/uk/gov/hmcts/reform/ccd/documentam/client/datastore/CaseDataStoreClient.java

```diff
@@ -1,11 +1,12 @@
-package uk.gov.hmcts.reform.ccd.documentam.service;
+package uk.gov.hmcts.reform.ccd.documentam.client.datastore;
 
 import uk.gov.hmcts.reform.ccd.documentam.model.DocumentPermissions;
 
 import java.util.Optional;
 import java.util.UUID;
 
-public interface CaseDataStoreService {
+public interface CaseDataStoreClient {
+    String EXPERIMENTAL_HEADER = "experimental";
```

Comment:
please remove this header - not required any more. I've cleaned up that in ccd-data-store a week ago..